pkg/minikube/machine/start.go

```diff
@@ -262,6 +282,21 @@ func DiskUsed(cr command.Runner, dir string) (int, error) {
 	return strconv.Atoi(percentage)
 }
 
+// DiskAvailable returns the available capacity of dir in the VM/container in GiB
+func DiskAvailable(cr command.Runner, dir string) (int, error) {
+	if s := os.Getenv(constants.TestDiskUsedEnv); s != "" {
+		return strconv.Atoi(s)
+	}
+	output, err := cr.RunCmd(exec.Command("sh", "-c", fmt.Sprintf("df -BG %s | awk 'NR==2{print $4}'", dir)))
+	if err != nil {
+		klog.Warningf("error running df -BG /var: %v\n%v", err, output.Output())
+		return 0, err
+	}
+	gib := strings.TrimSpace(output.Stdout.String())
+	gib = strings.Trim(gib, "G")
+	return strconv.Atoi(gib)
+}
+
 // postStart are functions shared between startHost and fixHost
```

Comment:
Function comment should begin like `// postStartSetup ...`

Reply:
good catch!

general question: that comment for example has nothing to do with the issue this PR refers to; is it advised to fix minor stuff like this whenever possible or should there be a separate cleanup issue (even though it doesn't change any logic)?